src/data/Dhis2Import.ts

```diff
@@ -1,106 +1,124 @@
 import _ from "lodash";
-import {
-    SynchronizationResult,
-    SynchronizationStats,
-    SynchronizationStatus,
-} from "../domain/entities/SynchronizationResult";
+import { SynchronizationResult, SynchronizationStats } from "../domain/entities/SynchronizationResult";
 import i18n from "../locales";
+import { D2Api } from "@eyeseetea/d2-api/2.33";
+import { Ref } from "../domain/entities/ReferenceObject";
 
-type Status = "OK" | "ERROR";
+export type Status = "OK" | "ERROR";
 
-export interface ImportPostResponse {
+type ImportStats = {
+    created: number;
+    deleted: number;
+    ignored: number;
+    updated: number;
+    total: number;
+};
+
+export type ImportPostResponse = {
+    status: Status;
+    response: Ref;
+};
+
+export type ImportReportResponse = {
     status: Status;
-    message?: string;
-    response?: {
-        status: SynchronizationStatus;
-        imported: number;
-        updated: number;
-        deleted: number;
-        ignored: number;
-        total: number;
-        importSummaries?: Array<{
-            responseType: "ImportSummary";
-            description?: string;
-            status: SynchronizationStatus;
-            href?: string;
-            importCount: {
-                imported: number;
-                updated: number;
-                deleted: number;
-                ignored: number;
+    validationReport: {
+        errorReports: [
+            {
+                uid: string;
+                message: string;
+            }
+        ];
+    };
+    stats: ImportStats;
+    bundleReport?: {
+        status: Status;
+        stats: ImportStats;
+        typeReportMap: {
+            [type: string]: {
```

Comment:
we can be more specific about the type by following the documentation https://docs.dhis2.org/en/full/develop/dhis-core-version-master/developer-manual.html

Something like this:
```ts
export type TrackerType = "TRACKED_ENTITY" | "EVENT" | "ENROLLMENT" | "RELATIONSHIP";

export type TypeReportMap = Record<
    TrackerType,
    { trackerType: TrackerType; stats: ImportStats; objectReports: { uid: Id } }
>;

```